### PR TITLE
Adding support for entity size limit

### PR DESCRIPTION
**Pull request by ladinu**

### CODE REVIEW

**Comment by stotten** [comment]
LGTM

File: src/main/scala/com/webtrends/harness/component/akkahttp/verbs/AkkaHttpEntity.scala [hunk 1]
```diff
@@ -13,8 +13,10 @@ trait AkkaHttpEntity[EntityT <: AnyRef] extends AkkaHttpBase {
 
   def unmarshaller: FromRequestUnmarshaller[EntityT] = AkkaHttpBase.unmarshaller[EntityT](ev)
 
+  def maxSizeBytes: Long = 1.024e6.toLong // 1MB
```
Comment (binarymechanic) [inline src/main/scala/com/webtrends/harness/component/akkahttp/verbs/AkkaHttpEntity.scala:16]: should this be configurable and why a def instead of val?

Comment (ladinu) [reply src/main/scala/com/webtrends/harness/component/akkahttp/verbs/AkkaHttpEntity.scala:16]: Yup. Any command that extend `AkkaHttpEntity[T]` can override `maxSizeBytes`

**Comment by binarymechanic** [comment]
looks good to me, too

**Pull request merged**